### ISSUE TITLE
QuestOD (UK catchup channel) some programmes are DRM protected

### ISSUE THREAD

**Issue by tarmacalastair** [Catch-up-TV-and-More/plugin.video.catchuptvandmore#177]
Hi,

Thanks for a great addon.

I've tried using QuestOD to view previously aired programmes. Some programmes on this channel work fine but the "Giro d'Italia Highlights" produces a message "_DRM Protected Video_". The same videos play back fine through Firefox on Ubuntu Linux.

Looking through some of the previous issues logged here, you seem to be able to fix some DRM issues. Is this one you can fix please?

Thanks for looking.

**Comment by wwark** [comment]
Hi @tarmacalastair,

What is the version of this add-on installed in your side ? The last "release" version is 0.2.6.
You can follow this documentation : https://catch-up-tv-and-more.github.io/installation/ to install the last version.

Thank you,
Wwark

**Comment by tarmacalastair** [comment]
Wow, fantastic, it's working great now - many thanks.

I've been running Kodi 17 (OSMC on a raspberry Pi) until now, with 0.1.something of the plugin. The other day when I tried QuestOD it simply said "DRM Protected Video" but today it prompted me to update to Kodi 18 which I did and then to install a DRM module from Google (Widevine I think). 

I did this and all is now working brilliantly. The plugin is now at v 0.2.6

What's great is that this channel is only available in SD quality over the air but the catchup is 720p so it's much clearer.

Thank again, excellent plugin.
